cli/azd/cmd/middleware/debug.go

```diff
@@ -10,7 +10,7 @@ import (
 func UseDebug() actions.MiddlewareFn {
 	return func(ctx context.Context, options *actions.ActionOptions, next actions.NextFn) (*actions.ActionResult, error) {
 		console := input.GetConsole(ctx)
-		console.Confirm(ctx, input.ConsoleOptions{
+		_, _ = console.Confirm(ctx, input.ConsoleOptions{
```

Comment:
is this planned to be part of the feature?

Reply:
This is mostly for debugging which allows devs an easy way to attach a VS code debugger to the running azd process before continuing.  Longer term I'd love to figure out a way to determine if we are in dev/debug mode and optionally include this middleware to support our development.